src/Microsoft.DotNet.Interactive.SQLite/ConnectSQLiteCommand.cs

```diff
@@ -25,9 +25,9 @@ public ConnectSQLiteCommand()
         InvocationContext commandLineContext)
     {
         var connectionString = commandLineContext.ParseResult.GetValueForArgument(ConnectionStringArgument);
-        var connector = new SQLiteKernelConnector(connectionString);
         var localName = commandLineContext.ParseResult.GetValueForOption(KernelNameOption);
-        var kernel = await connector.CreateKernelAsync(localName);
+        var kernel1 = new SQLiteKernel($"sql-{localName}", connectionString);
+        var kernel = await Task.FromResult<Kernel>(kernel1);
         return new[] { kernel };
```

Comment:
The await is a bit confusing. Why not remove the `async` keyword and say something like `return Task.FromResult(new [] { new SQLiteKernel($"sql-{localName}", connectionString) });`?

Reply:
That was a weird artifact of the inlining refactoring. Fixed.